src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/DebtorsController.java

```diff
@@ -0,0 +1,112 @@
+import java.util.List;
+
+@Controller
+@NextController(ReviewController.class)
+@PreviousController(OtherAccountingPolicyController.class)
+@RequestMapping("/company/{companyNumber}/transaction/{transactionId}/company-accounts" +
+        "/{companyAccountsId}/small-full/debtors")
+public class DebtorsController extends BaseController implements ConditionalController {
+
+    @Autowired
+    private DebtorsService debtorsService;
+
+    @Autowired
+    private BalanceSheetService balanceSheetService;
+
+    @Override
+    protected String getTemplateName() {
+        return "smallfull/debtors";
+    }
+
+    @GetMapping
+    public String getDebtors(@PathVariable String companyNumber,
+            @PathVariable String transactionId,
+            @PathVariable String companyAccountsId,
+            Model model, HttpServletRequest request) throws ServiceException {
+
+        addBackPageAttributeToModel(model, companyNumber, transactionId, companyAccountsId);
+
+        try {
+            Debtors debtors = debtorsService.getDebtors(transactionId, companyAccountsId,
+                    companyNumber);
+
+            model.addAttribute("debtors", debtors);
+
+        } catch (ServiceException e) {
+            LOGGER.errorRequest(request, e.getMessage(), e);
+            return ERROR_VIEW;
+
+        }
+
+        return getTemplateName();
+    }
+
+
+    @PostMapping
+    public String postDebtors(@PathVariable String companyNumber,
+            @PathVariable String transactionId,
+            @PathVariable String companyAccountsId,
+            @ModelAttribute("debtors") @Valid Debtors debtors,
+            BindingResult bindingResult,
+            Model model,
+            HttpServletRequest request) {
+
+        addBackPageAttributeToModel(model, companyNumber, transactionId, companyAccountsId);
+
+        if (bindingResult.hasErrors()) {
+            return getTemplateName();
+        }
+
+        try {
+            List<ValidationError> validationErrors =
+                    debtorsService.submitDebtors(transactionId, companyAccountsId,
+                            debtors, companyNumber);
+
+            if (! validationErrors.isEmpty()) {
+                bindValidationErrors(bindingResult, validationErrors);
+                return getTemplateName();
+            }
+        } catch (ServiceException e) {
+            LOGGER.errorRequest(request, e.getMessage(), e);
+            return ERROR_VIEW;
+        }
+
+        return navigator.getNextControllerRedirect(this.getClass(), companyNumber, transactionId,
+                companyAccountsId);
+    }
+
+    @Override
+    public boolean willRender(String companyNumber, String transactionId, String companyAccountsId) {
+        try {
+            BalanceSheet balanceSheet = balanceSheetService.getBalanceSheet(
+                transactionId, companyAccountsId, companyNumber);
+            return balanceSheet.getCurrentAssets() != null && balanceSheet.getCurrentAssets().getDebtors() != null;
```

Comment:
I believe the note should be shown if there is a value in the current or previous debtors on the balance sheet

Reply:
Ignore that!